src/rust-crypto/verification.ts

```diff
@@ -475,13 +480,21 @@ abstract class BaseRustVerifer<InnerType extends RustSdkCryptoJs.Qr | RustSdkCry
                 }
 
                 this.emit(VerificationRequestEvent.Change);
-            };
+            });
             inner.registerChangesCallback(onChange);
         });
         // stop the runtime complaining if nobody catches a failure
         this.completionPromise.catch(() => null);
     }
 
+    public setInner(inner: InnerType): void {
```

Comment:
given this is `public` (if only in this file), would be good to document it.

Reply:
done